src/generateRelative.ts

```diff
@@ -0,0 +1,96 @@
+import delve from "dlv";
+import { dset } from "dset";
+import { colors, generate, numberRanges } from "./generate";
+import { features, races, type FaceConfig, type Gender } from "./common";
+import { deepCopy } from "./utils";
+import { svgsGenders, svgsIndex } from "./svgs-index";
+
+// Currently, race just affects skin color and hair color. Let's ignore hair color (since you could imagine it being dyed anyway) and figure out someone's race just based on skin color. If no race is found, return undefined.
+const imputeRace = (face: FaceConfig) => {
+  return races.find((race) => colors[race].skin.includes(face.body.color));
```

Comment:
Probably a non-issue now (and likely ever), but if multiple races share a skin color, might have conflict w/ this function

Reply:
Yeah... it would also break if the default skin colors change (unless the old defaults are kept around for this function). But I think it's good enough for now, we just need to remember to deal with it if a future update introduces an issue.